Sources/SnapshotTesting/SnapshotTestCase.swift

```diff
@@ -81,7 +82,7 @@ open class SnapshotTestCase: XCTestCase {
 
       guard !recording, fileManager.fileExists(atPath: snapshotFileUrl.path) else {
         try strategy.diffable.to(diffable).write(to: snapshotFileUrl)
-        XCTFail("Recorded snapshot to \(snapshotFileUrl)", file: file, line: line)
+        XCTFail("Recorded snapshot: …\n\n\"\(snapshotFileUrl.path)\"", file: file, line: line)
```

Comment:
Makes it easier to double-click to select the file name and open.

Reply:
great UX!